pkg/controller/v1alpha1/inferencegraph/knative_reconciler.go

```diff
@@ -203,6 +203,30 @@ func createKnativeService(componentMeta metav1.ObjectMeta, graph *v1alpha1api.In
 											Drop: []v1.Capability{v1.Capability("ALL")},
 										},
 									},
+									VolumeMounts: []v1.VolumeMount{
+										{
+											Name:      "openshift-service-ca-bundle",
```

Comment:
Every pod already mount this file so you should not mount this. 
```/run/secrets/kubernetes.io/serviceaccount/service-ca.crt```

Reply:
Usually yes, but KServe has opted to set `AutomountServiceAccountToken` to false, which also has the side effect for not mounting the ca-bundle.

See https://github.com/opendatahub-io/kserve/blob/1382a8e60a96893d3914e533bc048caa27c2b6b8/pkg/controller/v1alpha1/inferencegraph/knative_reconciler.go#L209

Reply:
so now, IG need to access this cert so I think we should set this to true instead of adding the file.

Reply:
Flipping it to true will mount SA credentials which are not needed by IG. So, IMO, it is better to only mount the openshift bundle and skip the credentials.

Reply:
ok